pkg/skaffold/instrumentation/export.go

```diff
@@ -139,14 +139,16 @@ func createMetrics(ctx context.Context, meter skaffoldMeter) {
 	randLabel := label.String("randomizer", strconv.Itoa(rand.Intn(75000)))
 
 	m := global.Meter("skaffold")
+
+	// cloud monitoring only supports string type labels
 	labels := []label.KeyValue{
 		label.String("version", meter.Version),
 		label.String("os", meter.OS),
 		label.String("arch", meter.Arch),
 		label.String("command", meter.Command),
 		label.String("error", meter.ErrorCode.String()),
 		label.String("platform_type", meter.PlatformType),
-		label.Int("config_count", meter.ConfigCount),
```

Comment:
Ah makes sense. You are recording it as a label.
`helmCount` is recorded as a NewMetric.